src/main/java/com/hazelcast/commandline/HazelcastCommandLine.java

```diff
@@ -75,6 +77,13 @@ private static void runCommandLine(PrintStream out, PrintStream err, boolean sho
         }
     }
 
+    private static void checkIfEnvSupported() {
+        String osName = System.getProperty("os.name");
+        if (!osName.equals("Linux") && !osName.contains("OS X") && !osName.equals("SunOS") && !osName.equals("AIX")) {
```

Comment:
I think we should also check for "FreeBSD" and "Unix" (see https://memorynotfound.com/detect-os-name-version-java/)

Reply:
@googlielmo, we are using `java.lang.UNIXProcess` class to get pid of the process, and the supported environments by this class are specified like this. Please check `java.lang.UNIXProcess.Platform#get` method for details.

Reply:
I see, thanks for the reference!